aerospike/tests/common.py

```diff
@@ -4,7 +4,7 @@
 import os
 
 HERE = os.path.dirname(os.path.abspath(__file__))
-COMPOSE_FILE = os.path.join(HERE, 'docker', 'docker-compose.yaml')
+COMPOSE_FILE = os.path.join(HERE, 'docker', os.environ.get('DOCKER_COMPOSE_FILENAME'))
```

Comment:
```suggestion
COMPOSE_FILE = os.path.join(HERE, 'docker', os.environ.['DOCKER_COMPOSE_FILENAME'])
```
Let's let this fail fast if the envvar isn't defined.

Reply:
Accepted the review feedback